.env

```diff
@@ -1,7 +1,10 @@
 # MAIN SETTINGS FOR THE DOCKER COMPOSE STACK
 
 # Change this to the host name on which the stack is installed
-DJANGO_ALLOWED_HOSTS=localhost,127.0.0.1,192.168.99.110
+HOST=localhost
+
+# Change this to the host name on which the stack is installed
+DJANGO_ALLOWED_HOSTS=127.0.0.1,192.168.99.110
```

Comment:
is there a mean to move this to my setup only so we don't need to include the last ip?

Reply:
I've moved DJANGO_ALLOWED_HOSTS  to docker-compose.override only, so it is not used in production, meaning it's no big deal.

But we could remove the DJANGO_ALLOWED_HOSTS altogether if you set a HOST env var ? not sure how practical this is for you ? (we could maybe prefix it with SYLVIS_HOST, probably a good idea anyway)
Otherwise we could gitignore `.env` and just version a .env.example, so it's easier to make changes, it's one more command to get the stack up and running but that's not really a big deal.

Reply:
(if ok with you we can merge this and make that change separately)

Reply:
I like the idea of .env.example